Cargo.toml

```diff
@@ -15,4 +20,10 @@ async-trait = "0.1.50"
 either = "1.6.1"
 futures = "0.3.16"
 thiserror = "1.0.26"
+maplit = "1.0.2"
+
+[target.'cfg(target_os = "linux")'.dependencies]
 zbus = "1.9.1"
+
+[target.'cfg(target_os = "windows")'.dependencies]
```

Comment:
`cfg(windows)`

Reply:
So `cfg(windows)` and `cfg(unix)` are actually shortcuts for `cfg(target_family = "windows")`
There isn't a Linux family, that is just part of `cfg(unix)`.
Since we need to have `cfg(target_os = "linux")` it seemed more consistent to do the same for Windows

Reply:
Consistency has value, but it should not eclipse simpler and shorter alternatives. The repositories I am familiar with also tend to flavor `cfg(windows)` and so do I in my repositories.

```sh
# In Rust repository
$ rg 'cfg\(windows\)' | wc -l
110
$ rg 'cfg\(target_os\s*=\s*"windows"\)' | wc -l
7

# In tokio repository
$ $ rg 'cfg\(windows\)' | wc -l
45
$ rg 'cfg\(target_os\s*=\s*"windows"\)' | wc -l
0
```
Also in [getrandom](https://github.com/rust-random/getrandom/blob/30308ae845b0bf3839e5a92120559eaf56048c28/src/lib.rs#L206).